spec/tbd_tests_spec.rb

```diff
@@ -1,9 +1,4 @@
-begin
-  # Try to load from the tbd gem.
-  require "tbd"
-rescue LoadError
-  require_relative "../lib/tbd"
-end
+require "tbd"
```

Comment:
Again, specs run with a local TBD gem files (loaded in $LOAD_PATH) - no point looking elsewhere.